src/main/java/com/assure/movie/controller/MovieController.java

```diff
@@ -4,25 +4,27 @@
 import com.assure.movie.common.converter.MovieConverter;
 import com.assure.movie.dto.ActorDTO;
 import com.assure.movie.dto.MovieDTO;
-import com.assure.movie.dto.domain.BuilderMovieModel;
 import com.assure.movie.model.domain.Actor;
 import com.assure.movie.model.domain.Movie;
 import com.assure.movie.service.MovieService;
+import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import javax.validation.Valid;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * @author Israel Yasis
  */
 @RestController
+@Api(tags = "Movies")
 public class MovieController extends ApiController {
```

Comment:
Let's assume you will add _@RequestMapping("movies")_ here, how will this impact in the definition of your Rest endpoints in this class?

Reply:
In this case this will replace the request mapping of the parentClass(the parent class has the request mapping "api"), and the result for this particular endpoint will be. "movies/movies" not "api/movies"